### ISSUE TITLE
Cannot read property 'checkExpressionChange' of undefined

### ISSUE THREAD

**Issue by werts** [formly-js/ngx-formly#1054]
**I'm submitting a ...**  (check one with "x")
```
[ x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**

after my update, it got an error that: checkExpressionChange of undefined,since it works fine at version 4.6.2. and deeply, I find it wrong at code : 

FormlyForm.prototype.ngDoCheck = /**
     * @return {?}
     */
    function () {
        if ((/** @type {?} */ (this.options)).checkExpressionChange) {
            this.checkExpressionChange();
        }
    };
All forms that used ngx-formly get the same error.
**Expected behavior**

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 2.0.X

angular version 6.0.6 

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

windows 10 
chrome
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by werts** [comment]
It works fine since i rollback it to 4.6.3

**Comment by VWilm** [comment]
i have the same issue on angular 6

**Comment by aitboudad** [comment]
fixed in `4.6.6` version
